go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/google/gofuzz v1.2.0
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/iov-one/cosmos-sdk-crud v0.2.1
+	github.com/iov-one/cosmos-sdk-crud v0.2.3
```

Comment:
@LeCodeurDuDimanche , all the integration tests pass with a virgin genesis file, but can you confirm that there won't be a consensus failure if we have existing blockchain data?

Reply:
Yes, I can confirm that